### PR TITLE
Remove-XGB-update-toolchain

### PR DESCRIPTION
**Pull request by jdroenner**
- [ ] I added an entry to [`CHANGELOG.md`](CHANGELOG.md) if knowledge of this change could be valuable to users.

---

Here is a brief summary of what I did:

<TEXT>

### CODE REVIEW

**Comment by jdroenner** [comment]
[Update Backend](https://trello.com/c/plqW5QDH/1670-update-backend)

File: datatypes/src/collections/feature_collection.rs [hunk 1]
```diff
@@ -805,6 +805,7 @@ pub trait FeatureCollectionInfos {
     fn byte_size(&self) -> usize;
 }
 
+#[allow(dead_code)]
```
Comment (michaelmattig) [inline datatypes/src/collections/feature_collection.rs:808]: maybe pub use it in the mod to get rid of the warning? Or remove it if we never use, or plan to use it.

File: datatypes/src/operations/spatial_relation.rs [hunk 2]
```diff
@@ -3,8 +3,3 @@
 pub trait Contains<T> {
     fn contains(&self, other: &T) -> bool;
 }
-
-/// This method returns `true` iff does this `Self` intersect with the input `T`
```
Comment (michaelmattig) [inline datatypes/src/operations/spatial_relation.rs:7]: why is this no longer needed?

File: operators/src/engine/clonable_operator.rs [hunk 3]
```diff
@@ -74,6 +74,7 @@ pub trait CloneableInitializedVectorOperator {
 }
 
 /// Helper trait for making boxed `InitializedPlotOperator`s cloneable
+#[allow(dead_code)]
```
Comment (michaelmattig) [inline operators/src/engine/clonable_operator.rs:77]: maybe just pub use it like the other CLoneable... traits

File: operators/src/processing/raster_vector_join/aggregator.rs [hunk 4]
```diff
@@ -19,8 +19,10 @@ pub trait Aggregator {
 
     fn add_null(&mut self, feature_idx: usize);
 
+    #[allow(dead_code /*is used in tests*/)]
```
Comment (michaelmattig) [inline operators/src/processing/raster_vector_join/aggregator.rs:22]: would it make sense to move this to a separate trait?

File: services/src/pro/datasets/postgres.rs [hunk 5]
```diff
@@ -504,6 +504,7 @@ where
 }
 
 #[async_trait]
+#[allow(dead_code)]
```
Comment (michaelmattig) [inline services/src/pro/datasets/postgres.rs:507]: maybe delete this trait together with DatasetMetaData struct

File: datatypes/src/operations/spatial_relation.rs (hunk 2 above)
Comment (jdroenner) [reply datatypes/src/operations/spatial_relation.rs:7]: it is never used

File: operators/src/processing/raster_vector_join/aggregator.rs (hunk 4 above)
Comment (jdroenner) [reply operators/src/processing/raster_vector_join/aggregator.rs:22]: maybe. But then it is still never used, right?

Comment (michaelmattig) [reply operators/src/processing/raster_vector_join/aggregator.rs:22]: could be a trait implemented in a mod that is only compiled for tests

File: services/src/pro/datasets/postgres.rs (hunk 5 above)
Comment (jdroenner) [reply services/src/pro/datasets/postgres.rs]: [09346fc](https://github.com/geo-engine/geoengine/pull/973/commits/09346fcdb7960b7a52a8ecfce637354cb00950f8)

File: datatypes/src/collections/feature_collection.rs (hunk 1 above)
Comment (jdroenner) [reply datatypes/src/collections/feature_collection.rs]: [6849724](https://github.com/geo-engine/geoengine/pull/973/commits/684972467e851106b49dca727adc36c1d5595550)

File: datatypes/src/operations/spatial_relation.rs (hunk 2 above)
Comment (jdroenner) [reply datatypes/src/operations/spatial_relation.rs]: [6849724](https://github.com/geo-engine/geoengine/pull/973/commits/684972467e851106b49dca727adc36c1d5595550)

File: operators/src/engine/clonable_operator.rs (hunk 3 above)
Comment (jdroenner) [reply operators/src/engine/clonable_operator.rs]: [6849724](https://github.com/geo-engine/geoengine/pull/973/commits/684972467e851106b49dca727adc36c1d5595550)

File: operators/src/processing/raster_vector_join/aggregator.rs (hunk 4 above)
Comment (jdroenner) [reply operators/src/processing/raster_vector_join/aggregator.rs:22]: i guess it is ok tu use the fields directly in test

Comment (jdroenner) [reply operators/src/processing/raster_vector_join/aggregator.rs]: [18a463c](https://github.com/geo-engine/geoengine/pull/973/commits/18a463c28893769d7ab3de9e686998b2792ed907)

File: datatypes/src/operations/spatial_relation.rs (hunk 2 above)
Comment (michaelmattig) [reply datatypes/src/operations/spatial_relation.rs]: ?

**Review by michaelmattig — APPROVED** [review]

**Pull request merged**